mobile/android/app/build.gradle

```diff
@@ -16,11 +26,38 @@ android {
             ignoreAssetsPattern '!.svn:!.git:!.ds_store:!*.scc:.*:!CVS:!thumbs.db:!picasa.ini:!*~'
         }
     }
+    signingConfigs {
+        release {
+            keyAlias keystoreProperties['keyAlias']
+            keyPassword keystoreProperties['keyPassword']
+            storeFile file(keystoreProperties['storeFile'])
```

Comment:
Move `keystoreProperties.load(new FileInputStream(keystorePropertiesFile))` above this line.
Then, try to put this block inside an `if (buildType == 'release') {`